libethash-cuda/keccak.cuh

```diff
@@ -328,12 +331,22 @@ __device__ __forceinline__ void keccak_f1600_init(uint2* s)
 
 	/* iota: a[0,0] ^= round constant */
 	s[0] ^= vectorize(keccak_round_constants[23]);
+
+	for(uint32_t i=0; i<12; i++)
+	{
+	    state[i] = s[i];
+	}
 }
 
-__device__ __forceinline__ uint64_t keccak_f1600_final(uint2* s)
+__device__ __forceinline__ uint64_t keccak_f1600_final(uint2* state)
 {
+	uint2 s[25];
 	uint2 t[5], u, v;
 
+	for (uint32_t i = 0; i<12; i++)
```

Comment:
There are several for loops in this file that has the "for (uint32_t i = 0; i<?; i++)" style, should I re-format them all?